exo-go/src/process_helpers/index.go

```diff
@@ -34,7 +34,9 @@ func RunAndLog(dir string, env []string, log func(string), commandWords ...strin
 	}
 	process := NewProcess(commandWords...)
 	process.SetDir(dir)
-	process.SetEnv(env)
+	if len(env) > 0 {
+		process.SetEnv(env)
+	}
```

Comment:
It appears you can get the current env with `os. Environ()`. Thoughts on appending that with the supplied env?